frontend/src/metabase/visualizations/visualizations/Funnel.jsx

```diff
@@ -138,6 +138,7 @@ export default class Funnel extends Component {
       },
       getProps: transformedSeries => ({
         items: transformedSeries.map(s => s.card),
+        hasEditSettings: false,
```

Comment:
What do you think about naming this to be more in line with https://github.com/metabase/metabase/blob/a4255c5e749bfc3053f20d3c64010055ae349917/frontend/src/metabase/visualizations/visualizations/Funnel.jsx#L149

e.g. `hasBreakoutSettings` or something. So we know this isn't just a flag for anything, but only a flag specifically to this breakout settings.